enterprise_gateway/services/processproxies/yarn.py

```diff
@@ -64,7 +64,13 @@ def __init__(self, kernel_manager, proxy_config):
             from requests_kerberos import HTTPKerberosAuth
             auth = HTTPKerberosAuth()
 
-        self.resource_mgr = ResourceManager(service_endpoints=endpoints, auth=auth)
+        # export EG_CERT_BUNDLE environment variable as string, path to CA Bundle
+        # example:
+        # export EG_CERT_BUNDLE="/var/lib/cloudera-scm-agent/agent-cert/cm-auto-global_cacerts.pem"
+        cert_bundle = 'EG_CERT_BUNDLE'
+        verify_default = True
+        cert_path = os.getenv(cert_bundle, verify_default)
```

Comment:
I would prefer we move this to the top of the file, where the other environment variables are defined.  This is a basic convention throughout the repo.
```suggestion
        cert_path = os.getenv('EG_CERT_BUNDLE', True)
```
It might be good to add a comment that `cert_path` defaults to a boolean that can also represent a path.

Reply:
sure, addressing this following commit. let me know if that looks good.